pkg/login/social/generic_oauth.go

```diff
@@ -217,7 +223,8 @@ func (s *SocialGenericOAuth) UserInfo(client *http.Client, token *oauth2.Token)
 	var rawUserInfoResponse HttpGetResponse
 	var err error
 
-	if !s.extractToken(&data, token) {
+	if !s.extractToken(&data, token) || s.apiUrl != "" {
+		s.log.Debug("Fetching UserInfo ", s.apiUrl, token)
```

Comment:
It looks like this could log a valid token, which is generally a bad idea.

Reply:
Removed.